README.rst

```diff
@@ -108,154 +108,111 @@ Latest News
 Introduction
 ------------
 
-NVIDIA NeMo Framework is a generative AI framework built for researchers and pytorch developers
-working on large language models (LLMs), multimodal models (MM), automatic speech recognition (ASR),
-and text-to-speech synthesis (TTS).
-The primary objective of NeMo is to provide a scalable framework for researchers and developers from industry and academia
-to more easily implement and design new generative AI models by being able to leverage existing code and pretrained models.
+NVIDIA NeMo Framework is a scalable and cloud-native generative AI framework built for researchers and PyTorch developers working on `Large Language Models <nemo/collections/nlp/README.md>`_ (LLMs), `Multimodal Models <nemo/collections/multimodal/README.md>`_ (MMs), `Automatic Speech Recognition <nemo/collections/asr/README.md>`_ (ASR), `Text to Speech <nemo/collections/tts/README.md>`_ (TTS), and `Computer Vision <nemo/collections/vision/README.md>`_ (CV). It is designed to help you efficiently create, customize, and deploy new generative AI models by leveraging existing code and pre-trained model checkpoints.
```

Comment:
Please pull out the domain specific readmes at the end of the main readme. We have had questions about ASR features and models we support at ICASSP this year already due to the hiding of the ASR domain features inside if `nemo/collections/asr/README.md`.

Reply:
Revert back the Key Features link. In addition, this issue will be addressed on a separate PR.